guide/process/process-file.md

```diff
@@ -121,6 +133,22 @@ Reference the outputs of a previous step.
   }]
 }" :types="{}" />
 
+#### Example
+```yaml
+key: erc20-notification
+steps:
+  ...
+  - type: task
+    instanceHash: "H74Qqq8nT5JZ9GSJmuSWLN5benWZPkUb5pYcvQLsoZX"
+    taskKey: taskY
+    inputs:
+      inputA: "Input1 to the task"
+      # or
+      stepKey: taskX
+      key: taskZ
```

Comment:
This is a data of `inputA` so should be indented like
```
inputs: 
  inputA:
    stepKey: stepX
    key: taskX
```

This also needs to be updated in the complete example